bears/natural_language/LanguageToolBear.py

```diff
@@ -53,8 +55,12 @@ def run(self,
         # language_check being there.
         from language_check import LanguageTool, correct
 
+        for requirement in list(self.__class__.REQUIREMENTS):
```

Comment:
alternatively we can use a list instead of a set for the requirement or make the pythonimportrequirement a class level constant, that'd be far simpler